roles/kubernetes/control-plane/templates/kubeadm-config.v1beta3.yaml.j2

```diff
@@ -447,6 +447,9 @@ clusterDNS:
 {% for dns_address in kubelet_cluster_dns %}
 - {{ dns_address }}
 {% endfor %}
+{% if container_checkpoint_enabled is defined and container_checkpoint_enabled -%}
```

Comment:
The ContainerCheckpoint can add like the other feature_gates.

The code is not suitable here.